src/util/ui_message.h

```diff
@@ -26,7 +31,10 @@ class ui_message_handlert : public message_handlert
   ui_message_handlert(const class cmdlinet &, const std::string &program);
 
   ui_message_handlert()
-    : _ui(uit::PLAIN), time(timestampert::make(timestampert::clockt::NONE))
+    : _ui(uit::PLAIN),
+      time(timestampert::make(timestampert::clockt::NONE)),
+      out(std::cout),
```

Comment:
First, there seems to be no point of even having a `std::ostream` member as there is a single function that relies on it, and no way to setting it to any other value. Second, this has no place in a header file.

Reply:
No, `out` is used many, many times in the .cpp file. Eventually, assigning `out` should be the only use of `std::cout` in the codebase.
The implementation of the constructor must be moved back into the cpp file.